### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Olevsky-Ivan**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [my_favourite_films, marks,
+                                collection_of_coins],
+                    "immutable": [lucky_number, pi, one_is_a_prime_number,
+                                  name, profile_info]}
```
Comment (viktoria-rybenchuk) [inline app/main.py:22]: move each key on a new line

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]
You should make changes just in main.py. Please rollback changes in another files

**Comment by Olevsky-Ivan** [comment]
corrected code, maybe

**Comment by Olevsky-Ivan** [comment]
ALL?

**Comment by Olevsky-Ivan** [comment]
all?

**Comment by Olevsky-Ivan** [comment]
all?

**Comment by Olevsky-Ivan** [comment]
all

**Comment by Olevsky-Ivan** [comment]
open

**Comment by Olevsky-Ivan** [comment]
1

**Comment by Olevsky-Ivan** [comment]
1

**Review by Olevsky-Ivan — COMMENTED** [review]
all?

**Comment by Olevsky-Ivan** [comment]
d

**Comment by Olevsky-Ivan** [comment]
all

**Comment by Olevsky-Ivan** [comment]
1

**Comment by Olevsky-Ivan** [comment]
ф

**Comment by Olevsky-Ivan** [comment]
Якщо я нажму, то просто залишиться комент

**Comment by Olevsky-Ivan** [comment]
а

**Comment by Olevsky-Ivan** [comment]
fd

**Pull request closed without merging**